testing/acceptance_test.py

```diff
@@ -109,12 +109,12 @@ def test_skip():
         )
         assert result.ret == 1
 
-    def test_exitfail_waits_for_workers_to_finish(
+    def test_exitfirst_waits_for_workers_to_finish(
```

Comment:
`-x` is short for `--exitfirst`, not `--exitfail`. With the change in this PR, there are no passing tests - each worker hits the "global" limit failure limit of 1 locally, and exit.